test/test.js

```diff
@@ -3,95 +3,109 @@ var assert = require('assert');
 var getWindow = require('../');
 var getDocument = require('get-document');
 
+var inBrowser = typeof window !== 'undefined';
+
+// Zuul is configured to pass on to Browserify a configuration to exclude the
+// following module, 'jsdom' because it is only for testing in node. Browsers
+// have real dom
+var jsdom = !inBrowser ? require('jsdom') : null;
+
+var doc = inBrowser ? document : jsdom.jsdom();
+var win = inBrowser ? window : doc.parentWindow;
```

Comment:
Kind of taking the opposite approach here, checking whether we are in browser instead of in node.

Not using `node-detect` any more. Instead, checking on `typeof window`, like @TooTallNate suggests.